app/server/postgraphile/authenticationPgSettings.js

```diff
@@ -17,11 +23,11 @@ const authenticationPgSettings = (req) => {
       'jwt.claims.sub': '00000000-0000-0000-0000-000000000000',
       'jwt.claims.user_groups': groups.join(','),
       'jwt.claims.priority_group': priorityGroup,
-      role: NO_AUTH_POSTGRES_ROLE
+      role: 'ciip_guest'
```

Comment:
Why this change? The NO_AUTH flag is intended for developers, and should be using whatever postgres superuser they have in their environment

Reply:
Oooh I completely misunderstood what this did, I missed it when I refactored this method out of index.js. Fix incoming